tests/Vendor/Order/OrderServiceTest.php

```diff
@@ -77,27 +77,35 @@ public function testListOrders(): void
         $orders = $this->buildVendorOrderService()->listOrders();
 
         $this->assertContainsOnly(OrderSummary::class, $orders);
-        $this->assertCount(2, $orders);
+        $this->assertTrue(count($orders) >= 2);
 
-        $order = $orders[0];
-        $this->assertSame(5, $order->getOrderId());
+        // To fix random sort
+        usort($orders, function ($a, $b) {
+            return $a->getOrderId() <=> $b->getOrderId();
+        });
+
+        $order = array_shift($orders);
+        $this->assertSame(4, $order->getOrderId());
```

Comment:
pareil dans les autres tests ajoutés (si t'as la motiv tu peux changer les anciens)
```suggestion
        static::assertSame(4, $order->getOrderId());
```